app/src/main/java/org/kiwix/kiwixmobile/main/KiwixMainActivity.kt

```diff
@@ -79,4 +139,51 @@ class KiwixMainActivity : CoreMainActivity() {
       it.onNewIntent(intent, this)
     }
   }
+
+  override fun onNavigationItemSelected(item: MenuItem): Boolean {
+    when (item.itemId) {
+      R.id.menu_support_kiwix -> openSupportKiwixExternalLink()
+      R.id.menu_settings -> openSettingsActivity()
+      R.id.menu_help -> start<HelpActivity>()
+      R.id.menu_host_books -> start<ZimHostActivity>()
+      R.id.menu_history -> openHistoryActivity()
+      R.id.menu_bookmarks_list -> openBookmarksActivity()
+      else -> return false
+    }
+    return true
+  }
+
+  private fun openSupportKiwixExternalLink() {
+    val intent = Intent(
```

Comment:
This doesn't work and feels contradictory to DRY. Should it be moved to activity extensions or do you have any other sexier solution? Copied from `CoreReaderFragment`@`externalLinkPopup(Intent intent)`.

Reply:
Hmm, I guess we still alert people about going to an actual browser because they might be offline.

The EXTRA_EXTERNAL_LINK seems fairly pointless and the click listeners on the dialog are wrong as I think there should be 3.

An extension method on Uri `browserIntent()` might not be bad